### PR TITLE
Support TLS big buffers

### PR DESCRIPTION
**Pull request by dvdlevanon**

### CODE REVIEW

File: tap/tlstapper/bpf/openssl_uprobes.c [hunk 1]
```diff
@@ -156,9 +156,28 @@ static __always_inline void ssl_uprobe(void* ssl, void* buffer, int num, struct
 		return;
 	}
 	
+	struct ssl_info *infoPtr = bpf_map_lookup_elem(map_fd, &id);
 	struct ssl_info info = {};
 	
-	info.fd = -1;
+	if (infoPtr == 0) {
```
Comment (nimrod-up9) [inline tap/tlstapper/bpf/openssl_uprobes.c:162]: No definition for `NULL`?

File: tap/tlstapper/bpf/include/maps.h [hunk 2]
```diff
@@ -11,9 +11,10 @@ Copyright (C) UP9 Inc.
 #define FLAGS_IS_READ_BIT (1 << 1)
 
 #define CHUNK_SIZE (1 << 12)
-#define CHUNK_X8_SIZE (1 << 15)
 #define MAX_CHUNKS_PER_OPERATION (8)
 
+#define SSL_INFO_MAX_TTL_NANO (1000000000l * 60l)
```
Comment (nimrod-up9) [inline tap/tlstapper/bpf/include/maps.h:16]: How did you come to choose this value?

Comment (dvdlevanon) [reply tap/tlstapper/bpf/include/maps.h:16]: Its a minute in nano, it seems logical to wait maximum a minute between two ssl_read/write calls of the same message.

File: tap/tlstapper/bpf/include/maps.h [hunk 3]
```diff
@@ -10,6 +10,11 @@ Copyright (C) UP9 Inc.
 #define FLAGS_IS_CLIENT_BIT (1 << 0)
 #define FLAGS_IS_READ_BIT (1 << 1)
 
+#define CHUNK_SIZE (1 << 12)
+#define MAX_CHUNKS_PER_OPERATION (8)
+
+#define SSL_INFO_MAX_TTL_NANO (1000000000l * 60l)
```
Comment (nimrod-up9) [inline tap/tlstapper/bpf/include/maps.h:16]:
```suggestion
// One minute in nano seconds. Chosen by gut feeling.
#define SSL_INFO_MAX_TTL_NANO (1000000000l * 60l)
```

**Review by nimrod-up9 — APPROVED** [review]

**Pull request merged**